clang/include/clang/Driver/Options.td

```diff
@@ -3410,7 +3410,8 @@ def fno_experimental_isel : Flag<["-"], "fno-experimental-isel">, Group<f_clang_
   Alias<fno_global_isel>;
 def fveclib : Joined<["-"], "fveclib=">, Group<f_Group>,
   Visibility<[ClangOption, CC1Option, FlangOption, FC1Option]>,
-    HelpText<"Use the given vector functions library">,
+    HelpText<"Use the given vector functions library."
+             "Note: In clang -fveclib={ArmPL,SLEEF} implies -fno-math-errno">,
```

Comment:
There is a way to specify help separately for Clang. `HelpTextForVariants<[ClangOption,CC1Option]`